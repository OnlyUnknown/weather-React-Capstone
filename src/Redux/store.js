import { configureStore } from '@reduxjs/toolkit';
import mainReducer from './Slices/mainpageSlice';

const store = configureStore({
  reducer: {
    main: mainReducer,
  },
});

export default store;
