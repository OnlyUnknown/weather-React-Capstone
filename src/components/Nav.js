const Nav = () => (
  <>
    <ul>
      <li>Home</li>
      <li>Mic</li>
      <li>Setting</li>
    </ul>
  </>
);

export default Nav;
