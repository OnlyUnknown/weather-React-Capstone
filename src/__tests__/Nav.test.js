import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Nav from '../components/Details';
import store from '../redux/store';

describe('Component render testing', () => {
  it('Line page renders correctly', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Nav />
      </Provider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
