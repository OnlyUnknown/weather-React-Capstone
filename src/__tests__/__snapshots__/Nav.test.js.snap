// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component render testing Line page renders correctly 1`] = `
<DocumentFragment>
  Loading
</DocumentFragment>
`;
